firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -232,17 +242,32 @@ export class FirestoreBigQueryEventHistoryTracker
       const documentIdColExists = fields.find(
         (column) => column.name === "document_id"
       );
+      const paramsColExists = fields.find(
+        (column) => column.name === "path_params"
+      );
 
-      if (!documentIdColExists) {
+      if (!documentIdColExists && this.config.wildcardIds) {
         fields.push(documentIdField);
         await table.setMetadata(metadata);
-        logs.addDocumentIdColumn(this.rawChangeLogTableName());
+        logs.addNewColumn(this.rawChangeLogTableName(), documentIdField.name);
+      }
+      if (!paramsColExists) {
+        fields.push(documentPathParams);
+        await table.setMetadata(metadata);
+        logs.addNewColumn(
+          this.rawChangeLogTableName(),
+          documentPathParams.name
+        );
       }
     } else {
       logs.bigQueryTableCreating(changelogName);
+      const schema = { ...RawChangelogSchema };
+      if (this.config.wildcardIds) {
+        schema.fields.push(documentPathParams);
+      }
       const options: TableMetadata = {
         friendlyName: changelogName,
-        schema: RawChangelogSchema,
+        schema: schema,
```

Comment:
Not an issue coding wise, but this can now be set as the value is the same as the key name

```js
{
  friendlyName: changelogName,
  schema,
}
```